visualization/test/test-page/App.jsx

```diff
@@ -252,6 +252,7 @@ import { createTheme } from "@uifabric/styling";
                           this.generateRandomProbs.bind(this, classDimension)}
                         stringParams={{contextualHelp: this.messages}}
                         theme={theme}
+                        explanationMethod="pfi"
```

Comment:
What's going on here?

Reply:
this is the test page, just wanted to validate that all the explainers actually work. PFI was the last one I tested, we could make this a dropdown in the test page code if there were more aspects determined by this than just the text of one callout. It did help me catch that I'd left 'mimic' out of the dictionary. Also this is how the explanationMethod is passed in in the real code (eg, python widget or studio).